### PR TITLE
General dms information

### PR DESCRIPTION
**Pull request by carriewright11**
#### What changes are being implemented in this Pull Request?

Adding to the writing the DMS section to just provide general guidance about each section as a first draft. @vortexing can adjust as templates are created and we learn more. 

#### What was your approach?

Added info from the slide presentation and the website.

### Tell potential reviewers what kind of feedback you are soliciting.

Look OK?

### CODE REVIEW

File: 04-writing_a_DMS_plan.Rmd [hunk 1]
```diff
@@ -5,20 +5,96 @@ ottrpal::set_knitr_image_path()
...
+General requirements:
+
+- The plan must be **less than 2 pages**.
+- Initially DMS plans will not be made public, but they **may be made public** in the future, so do not include anything that might be confidential or proprietary in your plan.
+
+These following are the major sections of the plan:
+
+1. **Data type** - describe what data (amount and type) will be generated over the course of funding and what data will or will not be shared
+2. **Tools, software, and code** (note code is not required to be shared) - describe in what tools you intend to use to manage and analyze the data
+3. **Standards** - describe any standards that you might use for your data
```
Comment (avahoffman) [inline 04-writing_a_DMS_plan.Rmd:21]:
```suggestion
3. **Standards** - describe any standards that you might use for your data and metadata
```

File: 04-writing_a_DMS_plan.Rmd [hunk 2]
```diff
@@ -5,20 +5,96 @@ ottrpal::set_knitr_image_path()
...
+4. **Data preservation, access, timelines** - describe where the data will be made available and when
+5. **Access, distribution, reuse considerations** - describe how you have carefully considered any reasons that might limit sharing
+6. **Oversight** - describe who will manage compliance of the DMS plan
+    
+    
+##  Data Type
+
+A. Types and amount data
+
+- Describe what kind of data will be generated over the course of funding and the expected amounts of data for each.
```
Comment (avahoffman) [inline 04-writing_a_DMS_plan.Rmd:31]:
```suggestion
- Describe what kind of data will be generated over the course of funding. 
- Describe the expected amounts of data for each kind of data.
```

File: 04-writing_a_DMS_plan.Rmd [hunk 3]
```diff
@@ -5,20 +5,96 @@ ottrpal::set_knitr_image_path()
...
+
+B. Data Sharing
+
+- Describe what data from the project will shared or not shared and justify this decision.
+
+C. Metadata, other relevant data, and associated documentation
+
+- List information about metadata.
```
Comment (avahoffman) [inline 04-writing_a_DMS_plan.Rmd:39]:
```suggestion
- Briefly list metadata that will help with interpretation of your data.
```

File: 04-writing_a_DMS_plan.Rmd [hunk 4]
```diff
@@ -5,20 +5,96 @@ ottrpal::set_knitr_image_path()
...
+- List other possibly relevant data (that you think is not considered scientific).
+- List any relevant information about how you plan to gather the data (ex. study protocols and data collection instruments) that you plan to share to make it easier for others to reproduce your work.
+
+
+## Related Tools, Software and/or Code:
+
+- Describe any software or code you intend to use to process or analyze the data (provide the names of the tools and how they can be accessed).
```
Comment (avahoffman) [inline 04-writing_a_DMS_plan.Rmd:46]: Could we provide some examples ( I think we might need to get this from NIH) - e.g., R version 4.2.0 (link)

File: 04-writing_a_DMS_plan.Rmd [hunk 5]
```diff
@@ -5,20 +5,96 @@ ottrpal::set_knitr_image_path()
...
+- Describe the names of any data standards that will be used for your data/metadata to enable easier sharing are repeated use of your data.
+- Describe how these standards will be applied to your data.
+- If no standards exist, justify this.
+- This could include: 
+  - data formats 
+  - data dictionaries
+  - data identifiers
+  - definitions
+  - unique identifiers
+  - other data documentation
```
Comment (avahoffman) [inline 04-writing_a_DMS_plan.Rmd:59]:
```suggestion
  - other data documentation
  
<div class = "dictionary">

**Data Standard**  

An agreed upon set of rules that allow information to be shared and processed in a uniform and consistent manner. This includes representations, formats, and definitions of data. 

Standards vary by discipline. An extensive list of standards you can search can be found [here](https://fairsharing.org/search?fairsharingRegistry=Standard). An example standard is the [MIAME](https://fairsharing.org/FAIRsharing.32b10v) reporting guideline for microarray experiments.

</div>
```

File: 04-writing_a_DMS_plan.Rmd [hunk 6]
```diff
@@ -5,20 +5,96 @@ ottrpal::set_knitr_image_path()
...
+
+## Data Preservation, Access, and Associated Timelines
+
+A.  List where the data will be shared
+
+- Describe the repositories for sharing the scientific data, as well as the metadata.
+
+B. Methods to make scientific data findable and identifiable
+
+ - Describe what you will do to make the data more findable and identifiable. For example, perhaps using a persistent unique identifier or
```
Comment (avahoffman) [inline 04-writing_a_DMS_plan.Rmd:69]:
```suggestion
 - Describe what you will do to make the data more findable and identifiable. For example, you could use a persistent unique identifier or
```

File: 04-writing_a_DMS_plan.Rmd [hunk 7]
```diff
@@ -5,20 +5,96 @@ ottrpal::set_knitr_image_path()
...
+other standard indexing tool. [Zenodo](https://zenodo.org/) can be a useful option for creating digital object identifiers (DOIs). 
+
+C. Timeline of scientific data sharing:
+
+- Describe when the scientific data will be made available to others.
+  - Recall that data must be shared no later than the time of publication of findings or the end of the award (whatever comes first), however the NIH encourages th the data be shared as early as possible.
+  - Renewals can have updated DMS plan with a different timeline, but if not approved must use previous DMS date.
```
Comment (avahoffman) [inline 04-writing_a_DMS_plan.Rmd:76]:
```suggestion
  - Renewals can have an updated DMS plan with a different timeline, but if not approved must use previous DMS date.
```

**Review by avahoffman — APPROVED** [review]
Looks great! Added some minor suggestions :)

File: 04-writing_a_DMS_plan.Rmd (hunk 5 above)
Comment (carriewright11) [reply 04-writing_a_DMS_plan.Rmd:59]: Love it!

**Comment by carriewright11** [comment]
> Also I corrected some of the spelling stuff in #17

Yeah I thought so - thanks!

File: 04-writing_a_DMS_plan.Rmd (hunk 4 above)
Comment (avahoffman) [reply 04-writing_a_DMS_plan.Rmd:46]: I think it's fine to add this later, just something to think about

**Pull request merged**